src/frontend/apps/impress/src/features/docs/doc-header/components/ModalExport.tsx

```diff
@@ -151,6 +153,9 @@ export const ModalExport = ({ onClose, doc }: ModalExportProps) => {
                 </Text>
               );
             },
+            table: (block, t) => {
+              return <Table data={block.content} transformer={t} />;
+            },
```

Comment:
nit: `t` is often referenced in the project as translation, could be more specific to avoid confusion ?

```suggestion
            table: (block, transformer) => {
              return <Table data={block.content} transformer={transformer} />;
            },
```